content/projects/tdd/string-calculator/_index.md

```diff
@@ -209,10 +209,10 @@ add("1,2,3//;\n1000,1;2")
 
 ## Instructions For Reviewers
 
-- Look out for file structure. It might seem obvious, but people sometimes mess up the file structure of a TDD project. Refer to : **Set up your environment** which is at the top of this page.
+- Tests are nessesary for this project. Proper testing to be followed. If someone uses mocks and spies - that is incompetent.
```

Comment:
this part can be removed as it is in the project instructions: `Tests are nessesary for this project. `